### ISSUE TITLE
fft correct?

### ISSUE THREAD

**Issue by jiangzeen** [MariusVanDerWijden/gpusnarks#16]
hi，I have run the fft，but the result is error，maybe the finite field is error for ntt？

**Comment by MariusVanDerWijden** [comment]
Yes, this is a very old implementation and it might contain a few bugs. If you are looking for fast fft implementation in cuda, you should maybe look att he stuff starkware built
